lib/features/children/generosity_challenge/utils/generosity_challenge_helper.dart

```diff
@@ -33,7 +33,7 @@ class GenerosityChallengeHelper {
     bool isDebug,
   ) {
     final now = tz.TZDateTime.now(
-      tz.getLocation(isDebug ? 'Europe/Amsterdam' : 'America/Tulsa'),
```

Comment:
Just curious, why wouldn't we do local time on users device? 

Receiving the notification will then always happen at 4pm no matter where to the user is